server/servers/deploy/src/test/scala/com/prisma/deploy/specutils/DeployTestServer.scala

```diff
@@ -114,10 +116,38 @@ case class DeployTestServer()(implicit dependencies: DeployDependencies) extends
         )
         .recover {
           case error: QueryAnalysisError => error.resolveError
+          case e                         => sys.error(e.toString)
```

Comment:
What is this supposed to do? I think this should just discard the stack trace and we therefore lose information in the exception that is reported in a test.

Reply:
oversight from debugging, I removed it.